### PR TITLE
Improved agent and hooks test coverage

### PR DESCRIPTION
**Pull request by matthewloring**

### CODE REVIEW

**Comment by matthewloring** [comment]
@ofrobots PTAL

File: test/standalone/test-index.js [hunk 1]
```diff
@@ -121,4 +122,43 @@ describe('index.js', function() {
     assert.equal(typeof agent.addTransactionLabel, 'function');
   });
 
+  it('should allow start and end span calls when disabled', function() {
+    agent.stop();
+    var span = agent.startSpan();
```
Comment (ofrobots) [inline test/standalone/test-index.js]: Not a comment on your change, but should we stop returning a value from startSpan?

Comment (matthewloring) [inline test/standalone/test-index.js]: Yea, there are a lot of layers of the api. Hopefully some good documentation and examples in the readme will clear up some of the confusion!

**Comment by ofrobots** [comment]
LGTM.

**Pull request merged**